MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -57,6 +57,10 @@
       <Compile Include="VerifiableEnumerator.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net451' ">
+	  <Compile Include="UnenumerableReadOnlyList.cs" />
```

Comment:
We're not, but when I tried to remove it, I got this for the net451 build:
```
1) Error : MoreLinq.Test.SliceTests.TestSliceReadOnlyListOptimization
System.NotImplementedException : The method or operation is not implemented.
   at MoreLinq.Test.UnenumerableReadOnlyList`1.GetEnumerator() in C:\_SOURCECODE\MoreLINQ\MoreLinq.Test\UnenumerableReadOnlyList.cs:line 25
   at System.Linq.Enumerable.<SkipIterator>d__31`1.MoveNext()
   at System.Linq.Enumerable.<TakeIterator>d__25`1.MoveNext()
   at System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   at MoreLinq.Test.Enumerable.Count[TSource](IEnumerable`1 source) in C:\_SOURCECODE\MoreLINQ\MoreLinq.Test\Enumerable.cs:line 110
   at MoreLinq.Test.SliceTests.TestSliceReadOnlyListOptimization() in C:\_SOURCECODE\MoreLINQ\MoreLinq.Test\SliceTest.cs:line 142
```
If I had to guess, even though net451 implements netstandard1.2, the net40 version of MoreLinq is being pulled in because package resolution prefers implementation-specific types over falling back to netstandard.

The fix for this is to make the tests run in net40 (which would prevent issues like this happening again) or add a net451 target, but I don't think that needs to be fixed here; the mismatch is independent of #128.